src/pybullet_blocks/envs/cleanup_table_env.py

```diff
@@ -780,6 +845,154 @@ def is_toy_in_bin(self, toy_id: int) -> bool:
 
         return in_x_bounds and in_y_bounds and above_bottom and on_bottom
 
+    def is_object_ready_pick(self, object_id: int) -> bool:
```

Comment:
nice job figuring out something that works! we can revisit later if it looks too fake